DigitalLearningSolutions.Web/Views/NotificationPreferences/_NotificationPreferenceListContainer.cshtml

```diff
@@ -0,0 +1,25 @@
+﻿@using DigitalLearningSolutions.Web.Helpers

```

Comment:
Really minor thing, but if something is called container I'd expect it to take something to render inside it (e.g. like a layout does), whereas this seems to be a classic component. Maybe we could name it _UserNotificationPreferences.cshtml or something similar without a "Container" in it.